detect_test.go

```diff
@@ -94,4 +108,42 @@ func testDetect(t *testing.T, context spec.G, it spec.S) {
 			}))
 		})
 	}, spec.Sequential())
+
+	context("handles pyproject.toml parser", func() {
+		it("passes WorkingDir to the pyproject.toml parser #1", func() {
+			_, _ = detect(packit.DetectContext{
+				WorkingDir: "/hi",
+			})
+			Expect(parsePythonVersion.ParsePythonVersionCall.Receives.Path).To(Equal("/hi"))
+		})
+
+		it("passes WorkingDir to the pyproject.toml parser #2", func() {
+			_, _ = detect(packit.DetectContext{
+				WorkingDir: "/other",
+			})
+			Expect(parsePythonVersion.ParsePythonVersionCall.Receives.Path).To(Equal("/other"))
+		})
+
+		it("handles an error from the pyproject.toml parser #1", func() {
```

Comment:
The pattern we've used in other unit tests is to capture error handling in a separate context by itself at the bottom of the unit test.

I think we can have just one example of error handling inside of a new context for error handling - I don't think a second test adds any meaningful value because the chance of hard-coding the same arbitrary error in the code is very small.

Reply:
done